src/current/v23.1/connection-pooling.md

```diff
@@ -29,34 +29,90 @@ Connection pooling can be a enabled as a feature of the driver, a separate libra
 
+Configure the minimum number of connections to equal to the maximum number of connections, creating a fixed pool size.
+
+Set the maximum connection timeout value to 300 seconds, or 5 minutes. Do not set the connection pool timeout values to be too short, as it may cause the connection pool software to close and reopen connections frequently, causing increased latency. [Monitor the `sql.new_conns` metric](#monitor-new-connections) to make sure the timeout values are set correctly.
+
 {% include {{page.version.version}}/sql/server-side-connection-limit.md %} This may be useful in addition to your connection pool settings.
 
-## Validating connections in a pool
+</section>
+
+### Size connection pools in multi-region clusters
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+Similar to single-region {{ site.data.products.serverless }} clusters, the purpose of connection pools in multi-region {{ site.data.products.serverless }} clusters is to reduce connection latency. Add a connection pool for each region, and size the idle pool for your application and workload.
+
+The cluster will automatically scale up to meet demand, so there's no need to set a maximum connection pool size.
+
+{{ connection_timeout_value_serverless }}
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated selfhosted">
+
+For multi-region clusters, create a connection pool per region, and size the maximum connection pool for each region in your cluster using the same formula as a single-region cluster.
+
+For example, if you have 3 regions in your cluster, and each region has 12 vCPUs, create a connection pool for each region, with each connection pool having a maximum pool size of 48 (12 [processor cores] * 4).
+
+</section>
+
+## Validate connections in a pool
 
 After a connection pool initializes connections to CockroachDB clusters, those connections can occasionally break. This could be due to changes in the cluster topography, or rolling upgrades and restarts, or network disruptions. CockroachDB {{ site.data.products.cloud }} clusters periodically are restarted for patch version updates, for example, so previously established connections would be invalid after the restart.
 
 Validating connections is typically handled automatically by the connection pool. For example, in HikariCP the connection is validated whenever you request a connection from the pool, and the `keepaliveTime` property allows you to configure an interval to periodically check if the connections in the pool are valid. Whatever connection pool you use, make sure connection validation is enabled when running your application.
 
+<section class="filter-content" markdown="1" data-scope="dedicated selfhosted">
+
+## Monitor new connections
```

Comment:
This section applies to Serverless just as much as Dedicated/self-hosted - it should be made a common section for all of them.

Reply:
We don't expose `sql.new_conns` in the Cloud Console.

Reply:
Is it correct that the SQL Connection Attempts graph is how `sql.new_conns` is exposed?

Reply:
Yes, that's correct.